website/content/fr/wgs/platforms/glossary/_index.md

```diff
@@ -0,0 +1,43 @@
+
+## Utilisateurs de la Plateforme
+Les personnes qui utilisent directement les fonctionnalités de la plateforme, y compris, mais sans s'y limiter, les développeurs d'applications, les opérateurs d'applications, les scientifiques de données - quiconque déploie des applications sur la plateforme, utilise ses fonctionnalités ou nécessite des informations sur son utilisation. Les utilisateurs de la plateforme peuvent inclure d'autres ingénieurs de plateforme créant des services de plateforme de niveau supérieur en plus des fonctionnalités de niveau inférieur.
+
+## Portail
+Une interface Web qui fournit un accès centralisé à une variété de ressources, d'outils et de services. Il peut servir de point de départ à un large éventail d'utilisateurs afin de gérer et d'interagir efficacement avec les fonctionnalités de la plateforme sous-jacente. Un portail existe pour améliorer l'expérience utilisateur grâce à une interface conviviale qui simplifie les processus complexes et favorise les fonctionnalités de libre-service.
+
+## Fonctionnalités de la Plateforme
+Les avantages spécifiques pour les utilisateurs, ou **_qu'est-ce qu'_** une plateforme fournit. Celles-ci ne doivent pas être confondues avec les qualités de la plateforme qui décrivent **_comment_** les fonctionnalités s'utilisent. Ces fonctionnalités peuvent se situer à différents niveaux d'abstraction (par exemple, une base de données unique par rapport à un environnement de test comprenant une base de données) et fournies par différents fournisseurs de fonctionnalités. À mesure que les plateformes mûrissent, elles aspirent généralement à offrir des fonctionnalités en libre-service, en commençant par la possibilité de découvrir les fonctionnalités disponibles et en incluant une cohérence de l'expérience entre les fonctionnalités. Les fonctionnalités elles-mêmes sont souvent assez durables tandis que les fournisseurs et leur mise en œuvre peuvent évoluer plus rapidement. Par exemple, il est peu probable qu'une organisation cesse d'exiger des environnements de test, mais elle pourrait évoluer pour fournir des solutions conteneurisées au lieu de solutions basées sur des machines virtuelles.
+
+## Fournisseur de fonctionnalités de la Plateforme
+Un groupe de personnes qui développent et maintiennent une fonctionnalité offerte par la plateforme. Les fournisseurs peuvent être des organisations externes ou des équipes internes et, dans les petites organisations, il peut souvent s'agir des mêmes personnes qui développent également la plateforme plus large. À mesure que les plateformes mûrissent, elles bénéficient du maintien d'abstractions pour les fournisseurs afin de décourager le "verrouillage" et de continuer à progresser vers leur plateforme viable la plus petite (TVP).

```

Comment:
> d’abstractions pour les fournisseurs 
`d'abstractions vis à vis des fournisseurs`

> le "verrouillage"
je rajouterai (lock-in) derrière car parle beaucoup plus coté IT.